noty-android/app/src/main/java/dev/shreyaspatil/noty/view/viewmodel/HomeViewModel.kt

```diff
@@ -16,12 +16,18 @@
 
 package dev.shreyaspatil.noty.view.viewmodel
 
-import androidx.lifecycle.ViewModel
 import dagger.hilt.android.lifecycle.HiltViewModel
+import dev.shreyaspatil.noty.core.session.SessionManager
+import dev.shreyaspatil.noty.view.state.HomeState
 import javax.inject.Inject
 
-/**
- * Currently not in use
- */
 @HiltViewModel
-class HomeViewModel @Inject constructor() : ViewModel()
+class HomeViewModel @Inject constructor(
```

Comment:
I guess HomeFragment or its VM is no more usable. Think of removing this since its stale since long time

Reply:
I guess we can think of this sometime later